nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/src/main/java/org/apache/nifi/web/security/oidc/StandardOidcIdentityProvider.java

```diff
@@ -110,6 +117,11 @@ public void initializeProvider() {
             return;
         }
 
+        // Decide whether to use NiFi truststore instead of system's cacerts when connecting to OIDC provider
+        if (TruststoreStrategy.valueOf(properties.getOidcClientTruststoreStrategy()) == TruststoreStrategy.NIFI) {
```

Comment:
It looks like the latest automated build failed in relation to this enum.  To avoid potential parsing failures related to `TruststoreStrategy.valueOf()`, this could be changed to check against the string value of the property:
```suggestion
        if (TruststoreStrategy.NIFI.name().equals(properties.getOidcClientTruststoreStrategy())) {
```